indy_node/server/config_req_handler.py

```diff
@@ -60,14 +65,16 @@ def _doStaticValidationAuthRule(self, identifier, reqId, operation):
                                        "Transaction for change authentication "
                                        "rule for {}={} must not contain field {}".
                                        format(AUTH_ACTION, ADD_PREFIX, OLD_VALUE))
+        self._check_auth_key(operation, identifier, reqId)
 
-        auth_key = self.get_auth_key(operation)
-
-        if auth_key not in self.write_req_validator.auth_map and \
-                auth_key not in self.write_req_validator.anyone_can_write_map:
-            raise InvalidClientRequest(identifier, reqId,
-                                       "Unknown authorization rule: key '{}' is not "
-                                       "found in authorization map".format(auth_key))
+    def _doStaticValidationGetAuthRule(self, identifier, req_id, operation):
+        required_fields = list(dict(ClientGetAuthRuleOperation.schema).keys())
+        required_fields.remove(OLD_VALUE)
+        if len(operation) > 1:
+            if not set(required_fields).issubset(set(operation.keys())):
```

Comment:
Should we do similar checks as in `_doStaticValidationAuthRule` related to `OLD_VALUE `?

Reply:
I think, no. It's not a command which change something. If a request contains excess fields, but we can found a key maybe will be better to do it.